language/move-prover/src/bytecode_translator.rs

```diff
@@ -545,6 +550,17 @@ impl<'env> ModuleTranslator<'env> {
                 _ => {}
             }
         }
+        // (c) Walk over parameters and collect mutual references if they have update invariants.
```

Comment:
I recommend doing a replace everywhere of mutual -> mutable.

Reply:
It seems I've covered them.